doc/classes/Viewport.xml

```diff
@@ -350,23 +370,41 @@
 		<constant name="MSAA_16X" value="4" enum="MSAA">
 			Use 16x Multisample Antialiasing. Likely unsupported on medium and low-end hardware.
 		</constant>
+		<constant name="SCREEN_SPACE_AA_DISABLED" value="0" enum="ScreenSpaceAA">
+			Do not perform any antialiasing in the full screen post-process.
+		</constant>
+		<constant name="SCREEN_SPACE_AA_FXAA" value="1" enum="ScreenSpaceAA">
+			Use fast approximate antialiasing. FXAA is a popular screen space antialising method, it is very fast but low quality.
```

Comment:
```suggestion
			Use fast approximate antialiasing. FXAA is a popular screen-space antialising method, which is fast but will make the image look blurry, especially at lower resolutions. It can still work relatively well at large resolutions such as 1440p and 4K.
```